extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/security/AbstractFormAuthTestCase.java

```diff
@@ -84,7 +84,7 @@ public void testFormBasedAuthSuccess() {
                 .then()
                 .assertThat()
                 .statusCode(302)
-                .header("location", containsString("/admin"))
+                .header("location", containsString("/landing"))
```

Comment:
Weird...

Reply:
Yeah, I didn't change it lightly. From what I saw in the test, this is actually the right behavior. Given it's triggered by a REST Assured update, I suspect it was a bug in REST Assured.

Reply:
Our cookie uses `redirect.setMaxAge(0)`. I'm wondering how Restassured handles this case (if it does).